frontend/app/routes/protected/person-case/contact-information.tsx

```diff
@@ -74,7 +76,10 @@ export async function action({ context, request }: Route.ActionArgs) {
         emailAddress: v.optional(
           v.pipe(v.string(), v.trim(), v.email(t('protected:contact-information.error-messages.email-address-invalid-format'))),
         ),
-        country: v.picklist(VALID_COUNTRIES, t('protected:contact-information.error-messages.country-required')),
+        country: v.picklist(
+          getLocalizedCountries(lang).map(({ id }) => id),
```

Comment:
You don't have to get localized contries in this context.

```suggestion
          getCountries().map(({ id }) => id),
```

Reply:
will make this change in a separate PR since there's more PP data to load for this route.